torchvision/models/detection/_utils.py

```diff
@@ -489,6 +489,8 @@ def _topk_min(input: Tensor, orig_kval: int, axis: int) -> Tensor:
     Returns:
         min_kval (Tensor): Appropriately selected k-value.
     """
+    if not torch.jit.is_tracing():
+        return min(orig_kval, input.size(axis))
```

Comment:
You probably need a `# type: ignore[arg-type]` here and potentially you don't need one on line 496? worth a try 🤷‍♂️

Reply:
Or I could use `Union[int, Tensor]` as return type.

Reply:
Nah JIT won't let you do cool things like this. Try lying to it.